### ISSUE TITLE
Exporting video (Web, YouTube HD), getting an error 'audio codec not found'

### ISSUE THREAD

**Issue by ChileBob** [OpenShot/libopenshot-audio#154]
Exporting video (Web, YouTube HD), getting an error 'audio codec not found' - this worked in previous versions.

Works fine when exporting to FHD (MP4 h.264)

Ubuntu 22.04, all codecs installed via 'apt install ubuntu-restricted-extras', OpenShot v3.1.1, libopenshot 0.3.2, installed via flatpak on 1-Aug-2023

**Comment by approaching236** [comment]
I'm definitely still getting this:
```
Sorry, there was an error exporting your video: 
A valid audio codec could not be found for this file. for file /path/to/filename Session Mar 29 2024.mp4
```

I tried installing `ubuntu-restricted-extras`, but that didn't resolve the issue. Is there any sort of log I could get y'all?

**Comment by Colorjet3** [comment]
I just tested Exporting to Web | Youtube using the latest dev daily build AppImage and it worked for me in Ubuntu 22.04 lts.

I tested to all Youtube options (Youtube HD, Youtube Standard, and YouTube (2k, 4k, 8k)) without any issues.

1.  Go to openshot.org/download/#daily and download the latest dev daily build AppImage.
2.  Make it executable.
3.  Start OpenShot and try your export again.

**Comment by 18legar** [comment]
Yes. the solution is the one Colorjet3 wrote, I'd the same problem with Ubuntu, Antix and now MX linux, using the appimage gave no errors. thank you Colorjet3 .
